static/app/components/tooltip.tsx

```diff
@@ -116,215 +111,262 @@ function computeOriginFromArrow(
   }
 }
 
-class Tooltip extends React.Component<Props, State> {
-  static defaultProps: DefaultProps = {
-    position: 'top',
-    containerDisplayMode: 'inline-block',
-  };
-
-  state: State = {
-    isOpen: false,
-    usesGlobalPortal: true,
-  };
-
-  async componentDidMount() {
-    if (IS_ACCEPTANCE_TEST) {
-      const TooltipStore = (await import('sentry/stores/tooltipStore')).default;
-      TooltipStore.addTooltip(this);
+function getTooltipPortal(useGlobalPortal: boolean): HTMLElement {
+  // Use a single global portal element for tooltips
+  if (useGlobalPortal) {
+    let portal = document.getElementById('tooltip-portal');
+    if (!portal) {
+      portal = document.createElement('div');
+      portal.setAttribute('id', 'tooltip-portal');
```

Comment:
Do we need the id? It seems like we return the reference, so we can just rely on that?

Reply:
Yeah I believe we might use it in some tests